apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -127,25 +131,74 @@ public void Log_EmptyFormattedMessage()
         [Fact]
         public void Log()
         {
+            var message = "a {things} message with stuff";
```

Comment:
So what's doing the format-filling here? Is it:

- Our code?
- ASP.NET Core?
- Stackdriver?

(Basically I'm nervous if it's our code, as this kind of thing is easy to mess up in terms of security...)

Reply:
This is done by ASP.NET Core in the [Logger Extensions](https://github.com/aspnet/Logging/blob/release/2.0/src/Microsoft.Extensions.Logging.Abstractions/LoggerExtensions.cs)

We don't do any formatting at all.